qiita_db/artifact.py

```diff
@@ -154,11 +151,6 @@ def create(cls, filepaths, artifact_type, name=None, prep_template=None,
             should be provided and the parameter `prep_template` should not
             be provided.
 
-        The parameters `can_be_submitted_to_ebi` and
-        `can_be_submitted_to_vamps` defaults to false and they should be
```

Comment:
Would it still be useful to note that the `submitted_to_vamps` column will be set to False if the artifact_type can be submitted to vamps?